### PR TITLE
Start/Stop File share

### PR DESCRIPTION
**Pull request by gnehapk**
tendrl-bug-id: Tendrl/dashboard#449
- Start/Stop File Share Implementation
- Modified tooltip placement
- Called SetFlags() in error callback of logout

### CODE REVIEW

**Comment by r0h4n** [comment]
@gnehapk Please attach an issue for this PR

**Comment by gnehapk** [comment]
Addresses #449

**Comment by r0h4n** [comment]
@gnehapk @cloudbehl the issue id also needs to be in commit msg, How are you guys missing this info consistently?
we need below line in commit msg

tendrl-bug-id: Tendrl/dashboard#449

**Comment by gnehapk** [comment]
@r0h4n Added the github issue in the commit message.

**Comment by cloudbehl** [comment]
@gnehapk I can only see The tendrl bug Id in the commit message there is no text written. Also write the text that justifies this commit.

**Example:**
Start/Stop File Share Implementation

tendrl-bug-id: Tendrl/dashboard#449

File: src/modules/file-share/stop-file-share/stop-file-share.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    function StopFileShareController($rootScope, $scope, $state, selectedFileShare, fileShareStore) {
+
+        var vm = this;
+
+        vm.cancelModal = cancelModal;
+        vm.closeModal = closeModal;
+        vm.next = next;
+
+        vm.modalHeader = {
+            "title": "Stop File Share",
```
Comment (cloudbehl) [inline src/modules/file-share/stop-file-share/stop-file-share.js]: I think We should start replacing File Share with Volumes.

File: src/modules/file-share/stop-file-share/stop-file-share.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("StopFileShareController", StopFileShareController);
+
+    /*@ngInject*/
+    function StopFileShareController($rootScope, $scope, $state, selectedFileShare, fileShareStore) {
```
Comment (cloudbehl) [inline src/modules/file-share/stop-file-share/stop-file-share.js]: We will need to put the efforts again to replace fileshare with volume so please change it accordingly.

File: src/modules/file-share/stop-file-share/stop-file-share.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="tendrl-stop-file-share-modal-container">
+    <div>
+        <custom-modal-header title="vm.modalHeader.title" close="vm.modalHeader.close"></custom-modal-header>
+        <div class="modal-body row">
+            <div class="col-md-3 text-center">
+                <i class="pficon pficon pficon-warning-triangle-o"></i> 
+            </div>
+            <div class="col-md-8 desc">
+                Stopping this file share will make data inaccessible to these clients. Do you want to continue?
```
Comment (cloudbehl) [inline src/modules/file-share/stop-file-share/stop-file-share.html]: I think We should start replacing File Share with Volumes.

File: src/commons/factories/file-share-factory.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+(function () {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.service("fileShareFactory", fileShareFactory);
```
Comment (cloudbehl) [inline src/commons/factories/file-share-factory.js]: I think we need to rename factory also with volumeFactory.

File: src/modules/file-share/stop-file-share/stop-file-share.js (hunk 2 above)
Comment (gnehapk) [inline src/modules/file-share/stop-file-share/stop-file-share.js]: @cloudbehl Its better to change all Fileshare text to Volume at once only. As on some screens it will be volume and on some it will be File Shares, which will not look good. Need to send a separate PR for File Share --> Volume change.
